requirements.txt

```
@@ -8,6 +8,7 @@ torch>=1.1.0
 torchvision
 dominate>=2.3.5
 pandas
-keras
-tensorflow-gpu==1.14.0
+shapely
 scikit-image
+keras
+tensorflow-gpu
```

IIRC Tensorflow has been deprecating `tensorflow_gpu` / `tensorflow-gpu` since v 1.15. They have even gone to lengths unifying CuDNN implementations with CPU-only behaviour.

```suggestion
tensorflow>=2.0
```